Marlin/src/HAL/HAL_LINUX/include/Arduino.h

```diff
@@ -49,8 +49,8 @@ typedef uint8_t byte;
 #define PGM_P const char *
 
 // Used for libraries, preprocessor, and constants
-#define min(a,b) ((a)<(b)?(a):(b))
-#define max(a,b) ((a)>(b)?(a):(b))
+#define _min(a,b) ((a)<(b)?(a):(b))
+#define _max(a,b) ((a)>(b)?(a):(b))
```

Comment:
Marlin doesn't use any macros called `_min` or `_max` so these lines can probably be removed entirely.